flight/PiOS/Common/printf2.c

```diff
@@ -0,0 +1,633 @@
+/*******************************************************************************
```

Comment:
This is currently used by the OSD. Compared to the other printf in, it has floating point support. However, there are some weird bugs that can lead to a crash, so we should find a better solution at some point. c.f. 

https://github.com/BrainFPV/TauLabs/commit/acf781fd9977b9f86c39b48678af700efea5ee80